dwave/embedding/chain_strengths.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import math
+
+__all__ = ['uniform_torque_compensation']
+
+def uniform_torque_compensation(bqm, embedding=None, prefactor=1.414):
+    """Calculates chain strength using the RMS of the problem's quadratic biases.
```

Comment:
Probably better to have the first line by shorter. "Chain strength that attempts to compensate for torque that would break the chain"

Then put the "RMS" part in the body of the docstring